docs/user/ml/index.asciidoc

```diff
@@ -223,3 +223,9 @@ point type selector to filter the results by specific types of change points.
 
 [role="screenshot"]
 image::user/ml/images/ml-change-point-detection-selected.png[Selected change points]
+
+You can attach selected charts to a dashboard or a case by using the context 
```

Comment:
it's also possible to attach without selecting charts, so my suggestion is:

```
You can attach change point charts to a dashboard or a case by using the context menu. If the split field is selected, you can either select specific charts (partitions) or set the maximum number of top change points to plot. It's possible to preserve the applied time range or use the time bound from the page date picker.
You can also add or edit change point charts directly from the **Dashboard** app. 
```

To many **you can**, but the message is clear :)